qulice-checkstyle/src/test/resources/com/qulice/checkstyle/LocalVariableNames.java

```diff
@@ -7,6 +7,7 @@
  * Simple.
  * @version $Id$
  * @author John Smith (john@example.com)
+ * @checkstyle FinalLocalVariable (51 lines)
```

Comment:
@jrdalpra why you've added this suppression here? We are not testing suppressions here